cmd/registry/controller/controller.go

```diff
@@ -30,40 +31,48 @@ type ResourceCollection struct {
 func ProcessManifest(
 	ctx context.Context,
 	client connection.Client,
-	manifest *Manifest) ([]string, error) {
+	projectID string,
+	manifest *rpc.Manifest) ([]string, error) {
 
 	var actions []string
-	for _, entry := range manifest.Entries {
+	for _, resource := range manifest.GeneratedResources {
 
-		newActions, err := processManifestEntry(ctx, client, manifest.Project, entry)
+		newActions, err := processManifestResource(ctx, client, projectID, resource)
 		if err != nil {
-			log.Printf("Skipping entry: %q\nGot error: %s", entry, err.Error())
+			log.Printf("Skipping resource: %q\nGot error: %s", resource, err.Error())
 		}
 		actions = append(actions, newActions...)
 	}
 
 	return actions, nil
 }
 
-func processManifestEntry(
+func processManifestResource(
 	ctx context.Context,
 	client connection.Client,
-	project string,
-	entry ManifestEntry) ([]string, error) {
+	projectID string,
+	resource *rpc.GeneratedResource) ([]string, error) {
 	// Generate dependency map
-	resourcePattern := fmt.Sprintf("projects/%s/%s", project, entry.Resource)
-	dependencyMaps := make([]map[string]ResourceCollection, 0, len(entry.Dependencies))
-	for _, d := range entry.Dependencies {
-		dMap, err := generateDependencyMap(ctx, client, resourcePattern, d.Source, d.Filter)
+	resourcePattern := fmt.Sprintf("projects/%s/%s", projectID, resource.Pattern)
+	dependencyMaps := make([]map[string]ResourceCollection, 0, len(resource.Dependencies))
+	for _, d := range resource.Dependencies {
+		dMap, err := generateDependencyMap(ctx, client, resourcePattern, d.Pattern, d.Filter)
 		if err != nil {
-			log.Printf("Encountered error during updateResources().\n Error: %s\n Skipping entry %+v", err.Error(), entry)
+			log.Printf("Encountered error during generateDependencyMap().\n Error: %s\n Skipping resource %+v", err.Error(), resource)
```

Comment:
To avoid mismatched messages it might be a better to describe the situation, e.g. "error while generating dependency map" instead of using a specific function name that can be renamed globally without the message being noticed.

Reply:
Thanks for this. Done.